src/ui/shared/requests/useNativeBalance.ts

```diff
@@ -38,11 +38,13 @@ function useNativeAddressPosition({
           !item.protocol
       ) ?? [];
     if (nativePositions.length > 1) {
+      // eslint-disable-next-line no-console
       console.warn('multiple native positions');
     }
     return {
       data: nativePositions[0],
       isLoading,
+      isSuccess: Boolean(value?.positions),
```

Comment:
Hm, what about isDone flag? Can it be used here?

Reply:
I think `isSuccess` more familiar because of `react-query`